Backend/DinoTracker/Data/ResearcherRepository.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using System.Linq;
+using DinoTracker.Core;
+using Microsoft.EntityFrameworkCore;
+
+namespace DinoTracker.Data
+{
+    public class ResearcherRepository : IResearcherRepository
+    {
+        private DinoTrackerContext context;
+
+        public ResearcherRepository(DinoTrackerContext context)
+        {
+            this.context = context;
+        }
+
+        IEnumerable<Researcher> IResearcherRepository.GetAll()

```

Comment:
In the future, you can just put:

```
public IEnumerable<Researcher> GetAll()
{
    ...
}
```
And it will automatically know you're implementing that method in the interface.